pkg/frontend/openshiftcluster_preflightvalidation_test.go

```diff
@@ -20,6 +21,58 @@ import (
 func TestPreflightValidation(t *testing.T) {
 	ctx := context.Background()
 	mockSubID := "00000000-0000-0000-0000-000000000000"
+	preflightPayload := []byte(`
+	{
+		"apiVersion": "2022-04-01",
+		"id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resourceGroup/providers/Microsoft.RedHatOpenShift/openShiftClusters/resourceName",
+		"name": "resourceName",
+		"type": "microsoft.redhatopenshift/openshiftclusters",
+		"location": "eastus",
+		"properties": {
+			"clusterProfile": {
+			"domain": "default",
```

Comment:
nit: fix indentation in json to make it easier to read.

Reply:
Fixed the indentation